docs/user-guide/network-configuration/dhcp-vip-allocation.md

```diff
@@ -0,0 +1,19 @@
+# DHCP VIP Allocation
```

Comment:
Let's add a couple sentences as an intro:
1. What is it?
2. That it's not supported by OpenShift